lighthouse-core/gather/gatherers/inspector-issues.js

```diff
@@ -33,7 +33,9 @@ class InspectorIssues extends Gatherer {
   async beforePass(passContext) {
     const driver = passContext.driver;
     driver.on('Audits.issueAdded', this._onIssueAdded);
+    // try {
```

Comment:
we can remove these all now tho?

Reply:
It breaks appveyor rn :/ 
![image](https://user-images.githubusercontent.com/4071474/86855154-6967a080-c07f-11ea-9dc9-a4dc373e94ab.png)

but yeah once #10976 lands we can set a min chrome for the `is-on-https` expectation.